### DISCUSSION TITLE
remove locale / page under locale

### DISCUSSION THREAD

**Discussion by weber-s** [requarks/wiki#6782] [Help / Questions] [answered]
Hello!

I first created a wiki with language EN, but then switch to FR. I would like to mass delete all the page under the EN locale.

I did not find any bulk action button or "remove locale" action. Do I miss something or this feature is not yet implemented ? If so, can I do in another way (cli, db) ?

Thanks!

PS: I know there is also https://github.com/requarks/wiki/discussions/2994 but this is for url slug, that would be addressed in v3.

**Comment by NGPixel** [comment, marked as answer]
You would need to do it via the db.

Usual warning: do a backup first. Then run a query like:

```
DELETE FROM pages WHERE "localeCode" = 'en';
```
